libparsec/crates/protocol/tests/anonymous_account_cmds/v5/ping.rs

```diff
@@ -0,0 +1,63 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+// `allow-unwrap-in-test` don't behave as expected, see:
+// https://github.com/rust-lang/rust-clippy/issues/11119
+#![allow(clippy::unwrap_used)]
+
+use libparsec_tests_lite::prelude::*;
+
+use super::anonymous_account_cmds;
+
+// Request
+
+pub fn req() {
+    // Generated from Python implementation (Parsec v2.6.0+dev)
```

Comment:
![image](https://github.com/user-attachments/assets/50abf3d4-3332-4007-809e-4a6f075c9915)

Reply:
I regenerated it and it's still the same as for other ping commands ^^